stable/aws-for-fluent-bit/README.md

```diff
@@ -38,9 +38,12 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `imagePullSecrets` | Docker registry pull secret | `[]` |
 | `serviceAccount.create` | Whether a new service account should be created | `true` |
 | `serviceAccount.name` | Name of the service account | `aws-for-fluent-bit` |
-| `service.extraService` | Append to existing service with this value | `""` |
-| `service.parsersFiles` | List of available parser files | `/fluent-bit/parsers/parsers.conf` |
-| `service.extraParsers` | Adding more parsers with this value | `""` |
+| `service.type`| Type of service to be created - options are ClusterIP, NodePort, LoadBalancer |`ClusterIP`|
+| `service.port`| TCP port of the serviceMonitor service. | 2020 |
+| `service.targetPort`| TCP targetPort for service to connect to fluent-bit. | 2020 |
```

Comment:
@razorsk8jz is there any way to avoid this backwards breaking change? I will have to check into how EKS Charts should handle that.